osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -325,6 +316,124 @@ private double computeFlashlightValue(ScoreInfo score, OsuDifficultyAttributes a
             return flashlightValue;
         }
 
+        private double calculateComboBasedMissCount(ScoreInfo score, OsuDifficultyAttributes attributes)
```

Comment:
```suggestion
        private double calculateComboBasedMissCountEstimation(ScoreInfo score, OsuDifficultyAttributes attributes)
```

Reply:
Can we go with something like `calculateComboBasedEstimatedMissCount`? estimation post-fix reads weirdly to me

Reply:
That works too, my main idea is just to have word `estiamted` somewhere so that it doesn't look like its an actual miss calcualtion